common/bitutil/doc.go

```diff
@@ -10,5 +10,41 @@
 //
 // TODO-Klaytn: put the original LICENSE file in the common/bitutil directory
 
-// Package bitutil implements fast bitwise operations.
+/*
+Package bitutil implements fast bitwise operations and compression/decompressions
+
+Bitwise Operations
+
+Following operations are supported
+  - AND, OR, XOR operations
+  - Provides both safe version and fast version of above operations
+    `Safe` means it can be performed on all architectures
+    `Fast` means it only can be performed on architecture which supports unaligned read/write
+
+Compression and Decompression
+
+Following operations are supported
+  - CompressBytes
+  - DecompressBytes
+
+How compression works
```

Comment:
There is no `How decompression works`, can we have one?

Reply:
Basically decompression works in reverse way of compression. Anyway I'll try to add more description for decompression.